invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme.less

```diff
@@ -290,6 +290,14 @@ body {
 
 .ui.items > .item .extra.labels-actions {
   margin-bottom: 0.75em;
+
+  &.user-communities {
+    margin-top: 0.25em;
+
+    [class*="right floated"] {
```

Comment:
I think if you just define `.right.floated` if any other class is added, the rule will break, no? I wanted to avoid that!

Reply:
I don't think it works this way, otherwise no "btn btn-primary" in bootstrap would work, but correct me if I am wrong (see also https://frontstuff.io/should-you-chain-or-extend-css-classes#:~:text=more%20thoughtful%20choices.-,Chaining,with%20most%20modular%20CSS%20methodologies)
in this case I would be quite confused why any `.right.floated` element inside of `.user-communities` is supposed to have `margin-top:0;` is this the effect you want to get?
In general when I see a class accessed as an element attribute in css I am a bit alarmed, since there is the dot operator which should take care of it. We can also disscuss in the call